app/find_a_legal_advisor/routes.py

```diff
@@ -10,6 +10,9 @@ def search():
     form = FindLegalAdviserForm(request.args)
 
     category: str | None = request.args.get("category", default=None, type=str)
+    secondary_category: str | None = request.args.get(
```

Comment:
You are not validating the user's input here. This means they could enter whatever they want and it would get sent to LAALAA. Would it possible to validate this input please.